.github/workflows/test.yml

```diff
@@ -30,14 +30,9 @@ jobs:
       - name: install helm unittests
         run: |
           helm env
-          # Repeat 2 times to fight against network errors in  GHA runners
-          # Always return true
-          helm plugin install https://github.com/quintush/helm-unittest \
-            || helm plugin install https://github.com/quintush/helm-unittest \
-            || true
```

Comment:
So we're confident enough in GitHub CDN now? ^^
I would have done that in a separate PR ;)

Reply:
make sense, I was a bit enthusatsic :D

Reply:
https://github.com/jenkins-infra/helm-charts/pull/481/commits/dea3d318d6cbf16c42161decbfc50f7182daf768